lib/model.js

```diff
@@ -460,7 +465,7 @@ module.exports = function(registry) {
     if (ctx.result !== null) return cb();
 
     var fk = ctx.getArgByName('fk');
-    var msg = 'Unknown "' + toModelName + '" id "' + fk + '".';
+    var msg = g.f('Unknown "%s" id "%s".', toModelName, fk);
```

Comment:
Correct, and it's more structured and easier to read than the original, isn't it?